src/routes/customer-resource/customer-resource-show.js

```diff
@@ -145,22 +33,11 @@ class CustomerResourceShowRoute extends Component {
   }
 }
 
-function mapStateToProps(state) {
-  let { okapi, customerResourceShow } = state;
-  return {
-    okapi,
-    model: customerResourceShow || {
-      isLoaded: false,
-      isErrored: false
-    }
-  };
-}
-
-const loadRecord = (params) => ({
-  type: 'CUSTOMER_RESOURCE_SHOW_LOAD', params
-});
-const toggleSelected = (event) => ({
-  type: 'CUSTOMER_RESOURCE_SHOW_TOGGLE_SELECTED'
-});
-
-export default connect(mapStateToProps, { loadRecord, toggleSelected })(CustomerResourceShowRoute);
+export default connect(
+  ({ eholdings: { customerResource }}) => ({
```

Comment:
i'm not sure i fully understand this line, particularly the `eholdings` bit.  nearest i can guess is that this namespaces this component's state under an `eholdings` key?

Reply:
So with #52, I moved the `addReducer`/`addEpic` stuff to the root of our app under `this.addReducer('eholdings', rootReducer)`. This way when our components are initially mounted we have access to the default state and any other state other components may interact with.

Since `addReducer` adds the key `eholdings` to the state, all of our leaf reducers become part of that `eholdings` piece.

This line simply destructures that bit. Long form would be:

```js
(state) => {
  let customerResource = state.eholdings.customerResource
  ...
}
```

Reply:
👍 awesome.  that's much better than scattering the `addEpic` calls throughout the routes